### PR TITLE
Switch Mission equipment from duel selection to swap selection

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles/animationSources.hpp [hunk 1]
```diff
@@ -37,20 +37,19 @@ class AnimationSources
...
-    class msn_equip_american
+    class msn_equip_british
     {
-        displayName = "Install American Mission Equipment";
+        displayName = "Swap American Mission Equipment for British";
         author = "Apache mod development team";
-        onPhaseChanged = "";
+        forceAnimatePhase = 1;
+        forceAnimate[] = { "fcr_enable", 1};
+        onPhaseChanged = "if ((_this # 1) == 1) then {_this # 0 animatesource [""msn_equip_american"", 0];} else {_this # 0 animatesource [""msn_equip_american"", 1];};";
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles/animationSources.hpp:46]: Have you tried using the `forceAnimate` config field? https://community.bistudio.com/wiki/Arma_3:_Vehicle_Customization

**Review by mattysmith22 — COMMENTED** [review]
Have we tried that method I have described here?

**Comment by mattysmith22** [comment]
This worked for me:

```cpp
    class msn_equip_american
    {
        displayName = "Install American Mission Equipment";
        author = "Apache mod development team";
        source = "user";
        forceAnimatePhase = 1;
        forceAnimate[] = {"msn_equip_british", 0};
        initPhase = 1;
        animPeriod = 0.001;
    };
    class msn_equip_british
    {
        displayName = "Install British Mission Equipment";
        author = "Apache mod development team";
        source = "user";
        forceAnimatePhase = 1;
        forceAnimate[] = {"msn_equip_american", 0};
        initPhase = 0;
        animPeriod = 0.001;
    };
```

With the slight issue that you can have neither. But I guess if people want to die we can let them ...

**Comment by Ollieollieolllie** [comment]
No need to change then

**Pull request closed without merging**